packages/office-ui-fabric-react/src/components/Slider/Slider.types.ts

```diff
@@ -143,4 +149,5 @@ export interface ISliderStyles {
   activeSection: IStyle;
   inactiveSection: IStyle;
   valueLabel: IStyle;
+  zeroTick: IStyle;
```

Comment:
nit: I know it is not documented above, but it would seriously be nice to document this (describe what the "area" of the component the customization is for). There are many other components which fully document the IComponentStyle

Reply:
Documented all of them